### DISCUSSION TITLE
Non root context path for tolgee self hosted

### DISCUSSION THREAD

**Discussion by zeldigas** [tolgee/tolgee-platform#1809] [Q&A] [answered]
I'm trying to deploy Tolgee not in root of domain, smth like http://localhost:8080/tms/tolgee.

As tolgee is spring boot app, I'm adjusted `server.servlet.context-path` parameter to `/tms/tolgee` via env variables and it helped to get response from controller but static files are strill trying to load from root.

Is there additional parameter that can adjust paths for static content?

**Comment by JanCizmar** [comment, marked as answer]
This is not possible. On localhost, use another port. If you would like to publish your deployment, use subdomain.

**Reply by zeldigas** [reply]
Yes, localhost was just as playground. Any plans to support it in future? Or you target only dedicated domain deployment scheme?

**Reply by JanCizmar** [reply]
We don't plan to support this. Sorry. If you feel like that, you can submit a PR. :)
